pipeline/processors/labels.md

```diff
@@ -0,0 +1,104 @@
+# Labels
+
+The **labels** processor allows you to manipulate the labels of Metrics.
+
+Similar to the functionality exposed by filters, this processor presents a enriching/modifying mechanism to perform such operations for labels manipulation.
+The most significant difference is that processors perform better than filters, and when chaining them, there are no encoding/decoding performance penalties.
+
+Note that processors and this specific component can only be enabled using the new YAML configuration format. Classic mode configuration format doesn't support processors.
+
+## Configuration Parameters
+
+| Key         | Description |
+| :---------- | :--- |
+| update  | Update an existing key with a value into metrics. The key-value pair is required. |
+| insert  | Insert a new key with a value into metrics. The key-value pair is required. |
+| upsert  | Upsert a specific key with a value, the `upsert` operation will try to update the value of the key. If the key does not exist, the key will be created. The key-value pair is required. |
+| delete  | Delete a key from the labels of metrics. The key-value pair is required. |
+| hash    | Replace the key value with a hash generated by the SHA-256 algorithm from the specified label name, the binary value generated is finally set as an hex string representation. |
```

Comment:
Weirdly specific option - can we change the hash algorithm and other stuff in the future?

Reply:
Not sure. But I suppose that we shouldn't do this. Because there is no reason to use weaker hash algorithms such as SHA-1 or MD5.

Reply:
This should be used for anonymization of labels' values.